lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -19,10 +19,12 @@ function _storeInMDandDeleteData(bucketName, dataGetInfo, cipherBundle,
             if (err) {
                 return callback(err);
             }
+            let nullVersionLength;
             if (dataToDelete) {
+                nullVersionLength = dataToDelete[0].size;
```

Comment:
As discussed: handling metrics for null version overwrites will be introduced in a later PR.